wire-ios/Wire-iOS Tests/DeviceManagement/MockContextProvider.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+@testable import Wire
+import CoreData
+
+final class MockContextProvider: ContextProvider {
```

Comment:
Can't this mock be autogenerated by Sourcery?

Reply:
I tried generating. It failed to generate anything.

Reply:
Did you try to find out why it doesn't generate? I mean this sounds like a bug in Sourcery or our setup..

Reply:
Okay I found the problem: the sourcery run script was completely removed from `WireDataModel`, but the config and everything still existed. So it was a bug, that no generation happened at all.
I've fixed it here for this case on this brach, hope that's fine.

Reply:
I will probably take this up later